includes/class-hcard-user.php

```diff
@@ -317,7 +317,7 @@ public static function get_icon( $domain ) {
 		}
 
 		// Substitute another svg sprite file
-		$sprite = apply_filters( 'indieweb_icon_sprite', plugin_dir_url( __FILE__ ) . 'social-logos.svg', $domain );
+		$sprite = apply_filters( 'indieweb_icon_sprite', plugin_dir_url( __FILE__ ) . 'static/img/social-logos.svg', $domain );
```

Comment:
This is the only issue I see. Since this is called with plugin_dir_url, it will try to call /includes/static/img/social-logos.svg, which means it will break.